beacon-chain/state/state-native/getters_participation.go

```diff
@@ -39,6 +39,40 @@ func (b *BeaconState) PreviousEpochParticipation() ([]byte, error) {
 	return b.previousEpochParticipationVal(), nil
 }
 
+// CurrentEpochParticipationNoCopy corresponding to participation bits on the beacon chain without copying the data.
+// Modifications to the returned slice will modify the underlying data.
+func (b *BeaconState) CurrentEpochParticipationNoCopy() ([]byte, error) {
```

Comment:
From @nisdas 
```
CurrentEpochParticipationNoCopy and PreviousEpochParticipationNoCopy makes me nervous
we don't have a NoCopy method anywhere in our state, you could unintentionally mutate our state this way
since we have shared field refrences here
Maybe something like CurrentEpochParticipationReadOnly is returned and you do not have access to the underlying slice but some wrapper which allows you to read the underlying value, et
```